sdk/core/Azure.Core.Experimental/tests/LowLevelClient/PetStoreClient.cs

```diff
@@ -139,7 +162,29 @@ private HttpMessage CreateGetPetRequest(string id, RequestOptions options = null
             uri.AppendPath(id, true);
             request.Uri = uri;
             request.Headers.Add("Accept", "application/json, text/json");
+            message.ResponseClassifier = _responseClassifierCache["GetPet"];
```

Comment:
I imagine each method would be able to look up their response classifier by their method name.